uberfire-project/uberfire-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/utils/configuration/ConfigurationKey.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.common.services.project.backend.server.utils.configuration;
+
+/**
+ * Keys use to configure the Maven compiler in the kie-wb-common
+ *
+ * <p>
+ *  COMPILER jdt or javac
+ *  SOURCE_VERSION  configured with jvm source code version
```

Comment:
What's the expected version format (1.8 versus 8)? Which versions do we want to support?
Please add an example of the expected version format to the javadoc.

Reply:
1.8,  support older version requires tests and the availability of versions currently not freely available. I'll add the current values in the javadoc.